tests/performance/test_api/test_api.py

```diff
@@ -23,7 +38,9 @@ def test_api_endpoints(test_case, test_configuration, set_api_test_environment,
     """Make an API request for each `test_case`. `test_configuration` fixture is only used to add metadata to the
```

Comment:
After talking with @snaow we have considered that new development or changes into existing tests should follow new repository standards about Wazuh daemons. For now, it is required to use fixtures to start only required daemons, and turn them off when the test is over.  That means, that the default status of Wazuh would be off and each test should be responsible for daemons handling.
In case of the need of all Wazuh daemon, the test should disable every module not necessary for the testing in `ossec.conf`.

The reasons for this change are:
- Faster testing
- More encapsulated tests